experimental/dds/tree2/src/core/rebase/utils.ts

```diff
@@ -162,12 +162,25 @@ export function rebaseBranch<TChange>(
 	// Figure out how much of the trunk to start rebasing over.
 	const targetRebasePath = targetPath.slice(0, newBaseIndex + 1);
 	const newSourceCommits: GraphCommit<TChange>[] = [...targetRebasePath];
+	/** The commit on the target branch that the new source branch branches off of */
```

Comment:
Is this the same as the closest common ancestor?

Reply:
Yes

Reply:
I added a clarifying statement